faketests/slowtests/test_1.py

```diff
@@ -0,0 +1,33 @@
+import time
```

Comment:
how about using `pytest.mark.parametrize` there - as far as i can tell in all cases a range(1,9) is a fit parameterset